src/UI/WB.UI.Headquarters.Core/Code/WebInterview/Pipeline/PauseResumePipelineModule.cs

```diff
@@ -13,28 +15,36 @@ public class PauseResumePipelineModule : IPipelineModule
     {
         private readonly ICommandService commandService;
         private readonly IAuthorizedUser authorizedUser;
+        private readonly IStatefulInterviewRepository statefulInterviewRepository;
 
-        public PauseResumePipelineModule(ICommandService commandService, IAuthorizedUser authorizedUser)
+        public PauseResumePipelineModule(ICommandService commandService, IAuthorizedUser authorizedUser, 
+            IStatefulInterviewRepository statefulInterviewRepository)
         {
             this.commandService = commandService;
             this.authorizedUser = authorizedUser;
+            this.statefulInterviewRepository = statefulInterviewRepository;
         }
 
         public Task OnConnected(Hub hub)
         {
             var interviewId = hub.GetInterviewId();
-
+            
             if (authorizedUser.IsInterviewer)
             {
                 commandService.Execute(new ResumeInterviewCommand(interviewId, this.authorizedUser.Id,
                     AgentDeviceType.Web));
-
             }
             else if (authorizedUser.IsSupervisor)
             {
                 commandService.Execute(new OpenInterviewBySupervisorCommand(interviewId, this.authorizedUser.Id,
                     AgentDeviceType.Web));
             }
+            else

```

Comment:
if interview will be opened by any user (hq, admin, ...) interview will add ResumeInterviewCommand or PauseInterviewCommand

Reply:
logic was changed